### PR TITLE
Update Zoloto to 0.6.0

### PR DESCRIPTION
**Pull request by trickeydan**
As Zoloto 0.5.2 also includes inline type stubs :tada:, we can remove our stubs at last!

However, we need to install Zoloto in CI, and Zoloto does not support Python 3.8. 

This is heavily related to https://github.com/j5api/j5/issues/402

### CODE REVIEW

**Comment by trickeydan** [comment]
PR made upstream https://github.com/RealOrangeOne/zoloto/pull/163

File: pyproject.toml [hunk 1]
```diff
@@ -21,7 +21,7 @@ python = "^3.6"
 pyusb = "^1.0"
 pyserial = "^3.4"
 pyquaternion = "^0.9.5"
-zoloto = {version = ">=0.5,<0.6", optional = true}
+zoloto = {version = ">=0.5.2,<0.6", optional = true}
```
Comment (trickeydan) [inline pyproject.toml]: Also no. We're fine to use 0.5.1 or 0.5.0 here, as long as we use 0.5.2 or greater in dev.

**Comment by trickeydan** [comment]
@RealOrangeOne Do we have a timescale for adding 3.8 support upstream? It feels quite out of scope for this issue to keep implementing the changes needed to support new major versions of Zoloto. Also gets annoying as it diverges further from j5 `master`.

**Comment by RealOrangeOne** [comment]
@trickeydan undefined, but I might get a chance to look into it further over the weekend. The main blocker is the CV update

**Comment by trickeydan** [comment]
Closing for now to remove branch on main repo. See #571

**Pull request closed without merging**